modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_entries_admin/js/PersonAccountEntryEventHandler.es.js

```diff
@@ -49,8 +39,13 @@ class PersonAccountEntryEventHandler extends PortletBase {
 	 */
 	detached() {
 		super.detached();
-		this.selectUserButtonHandle_.removeListener();
-		this.removeUserButtonHandle_.dispose();
+
+		this.selectUserButton.removeEventListener(
+			'click',
+			this._handleSelectUserButtonClicked
```

Comment:
Same issue as noted elsewhere, `this._handleSelectUserButtonClicked !== this._handleSelectUserButtonClicked.bind(this)`, so this won't actually remove the listener.